metadata-ingestion/src/datahub/ingestion/source/feast.py

```diff
@@ -91,6 +96,9 @@ class FeastRepositorySourceConfig(ConfigModel):
     environment: str = Field(
         default=DEFAULT_ENV, description="Environment to use when constructing URNs"
     )
+    owner_mappings: List[Dict[str, str]]  = Field(
+        default={}, description="Mapping of owner names to owner types"
```

Comment:
As mentioned below, we should have a better description of this argument with mentions of which keys are required, what happens if keys are not specified, etc.

Reply:
I added the comment and also added owner_mappings as Optional